src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -186,8 +165,14 @@ async function modifyPermalinks(repoPath: string) {
       );
       let permalinkLineTrimmed = permalinkLine.trim();
 
-      // check if permalink has quotation marks, if so, remove them
-      permalinkLineTrimmed = permalinkLineTrimmed.replace(/"/g, "");
+      // check if permalink has quotation marks at the start and at the end, and if so, remove them
+      assert(permalinkLineTrimmed.startsWith(`permalink:`));
+      let permalinkValue = permalinkLineTrimmed
+        .replace(`permalink:`, "")
+        .trim();
+      permalinkValue = permalinkValue.replace(/^"/g, "").replace(/"$/, "");
```

Comment:
Fix for bug 2, rather than doing a replace, we do a replace iff it starts with a quotation and ends with a quotation